pkgs/development/python-modules/python-ndn/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pythonOlder
+, setuptools
+, pycryptodomex
+, lark
+, pygtrie
+, aenum
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-ndn";
+  version = "0.3-1";
```

Comment:
There are no git tags, but according to PyPI, it looks like the latest release, 0.3-3, is [this commit](https://github.com/named-data/python-ndn/commit/8a852659139e974d7a624b40a9145b4844c9af07).